libcodechecker/session_manager.py

```diff
@@ -274,6 +285,9 @@ def create_or_get_session(self, auth_string):
             self.__cleanup_sessions()
 
         if self.__handle_validation(auth_string):
+            # If the session is still valid
```

Comment:
Could we constrict this comment to be longer and less line-breaks, if it still conforms to pep8?